src/main/scala/com/fulcrumgenomics/fastq/FastqToBam.scala

```diff
@@ -96,6 +96,7 @@ class FastqToBam
   @arg(          doc="Read group ID to use in the file header.")                               val readGroupId: String = "A",
   @arg(          doc="The name of the sequenced sample.")                                      val sample: String,
   @arg(          doc="The name/ID of the sequenced library.")                                  val library: String,
+  @arg(          doc="Library or Sample barcode sequence.")                                    val barcode: Option[String] = None,
```

Comment:
Thinking about this further, how would you feel about making this a `Seq[String]` with default `Seq.empty`, then joining them with a hyphen?  That way folks don't have to do so themselves?
```suggestion
  @arg(          doc="Library or Sample barcode sequence.", minElements=0)                                    val barcode: Seq[String] = Seq.empty,
```